cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/DcLineSegmentConversion.java

```diff
@@ -101,19 +101,19 @@ private static double getMaxP(double pAC1, double pAC2, HvdcLine.ConvertersMode
         return DEFAULT_MAXP_FACTOR * Math.abs(pAC1);
     }
 
-    private static double getPDc(double pAC1, double pAC2, double poleLossP1, double poleLossP2,
-        HvdcLine.ConvertersMode mode) {
+    private static double getActivePowerSetpoint(double pAC1, double pAC2, double poleLossP1, double poleLossP2,
```

Comment:
Maybe add a comment saying that the active power set point of an HVDC should be interpreted on AC side of the rectifier converter